ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineContext.java

```diff
@@ -132,7 +132,7 @@ synchronized EngineContext createSandbox(final ServiceContext serviceContext) {
     this.runtimeAssignor.rebuildAssignment(queryRegistry.getPersistentQueries().values());
     return new EngineContext(
         SandboxedServiceContext.create(serviceContext),
-        processingLogContext,
+        ProcessingLogContext.create(),
```

Comment:
Need to use the empty create or otherwise we're registering metrics when sandbox executing queries